src/pet1.js

```diff
@@ -0,0 +1,81 @@
+const MAXIMUM_FITNESS = 10;
```

Comment:
Most of the comments regards logic given in pet.js also apply here, would be good to update this in the same ways as in that file